paimon-web-server/src/main/java/org/apache/paimon/web/server/controller/JobController.java

```diff
@@ -0,0 +1,120 @@
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/** Job submit api controller. */
+@Slf4j
+@RestController
+@RequestMapping("/api/job")
+public class JobController {
+
+    @Autowired private JobService jobService;
+
+    @SaCheckPermission("playground:job:submit")
+    @PostMapping("/submit")
+    public R<JobVO> submit(@RequestBody JobSubmitDTO jobSubmitDTO) {
+        try {
+            return R.succeed(jobService.submitJob(jobSubmitDTO));
+        } catch (Exception e) {
+            log.error("Exception with submitting a job.", e);
+            return R.failed(Status.JOB_SUBMIT_ERROR);
+        }
+    }
+
+    @SaCheckPermission("playground:job:fetch")
+    @PostMapping("/fetch")
+    public R<ResultDataVO> fetchResult(@RequestBody ResultFetchDTO resultFetchDTO) {
+        try {
+            return R.succeed(jobService.fetchResult(resultFetchDTO));
+        } catch (Exception e) {
+            log.error("Exception with fetching result data.", e);
+            return R.failed(Status.RESULT_FETCH_ERROR);
+        }
+    }
+
+    @SaCheckPermission("playground:job:list")
+    @GetMapping("/list")
+    public R<List<JobVO>> list() {
+        return R.succeed(jobService.listJobs());
+    }
+
+    @SaCheckPermission("playground:job:list")
+    @GetMapping("/list/page")
+    public R<List<JobVO>> listJobsByPage(int current, int size) {
+        return R.succeed(jobService.listJobsByPage(current, size));
+    }
+
+    @SaCheckPermission("playground:job:query")
+    @GetMapping("/status/get/{jobId}")
+    public R<JobStatusVO> getJobStatus(@PathVariable("jobId") String jobId) {
+        JobInfo job = jobService.getJobByJobId(jobId);
+        JobStatusVO jobStatusVO =
+                JobStatusVO.builder().jobId(job.getJobId()).status(job.getStatus()).build();
+        return R.succeed(jobStatusVO);
+    }
+
+    @SaCheckPermission("playground:job:query")
+    @GetMapping("/statistics/get")
+    public R<JobStatisticsVO> getJobStatistics() {
+        return R.succeed(jobService.getJobStatistics());
+    }
+
+    @SaCheckPermission("playground:job:stop")
+    @PostMapping("/stop")
+    public R<Void> stop(@RequestBody StopJobDTO stopJobDTO) {
+        try {
+            jobService.stop(stopJobDTO);
+            return R.succeed();
+        } catch (Exception e) {
+            log.error("Exception with stopping a job.", e);
+            return R.failed(Status.JOB_STOP_ERROR);
+        }
+    }
+
+    @SaCheckPermission("playground:job:refresh")
+    @PostMapping("/refresh")
+    public R<Void> refresh() {
+        jobService.refreshFlinkJobStatus();
```

Comment:
Why does this invoke this `refreshFlinkJobStatus`? For `JobService`, it could not distingish any engine job.

Reply:
Currently there is only the flink engine, which will be modified later. Other engines don’t know how to obtain the status yet.

Reply:
@s7monk, I don't think that the interface should follow the engine implementaion. Please change to `refreshJob`.

Reply:
done.